test/lib/oauth.js

```diff
@@ -382,7 +397,8 @@ var setupAppConfig = function(config, callback) {
 
     var dummy = {
         close: function() {
-            child.kill();
+            child.kill("SIGKILL"); // SIGKILL really needed??
```

Comment:
I can test it again without the sigkill. This is in the test/lib directory, it's a helper for setting up the app for testing. It seemed not the most important part of code to spend a lot of time on. Normal pump servers don't start stop every few seconds. With testing they can/do. So the 'extreme' SIGKILL will make sure every thing is finished.

This can be a case for further investigation, but I don't want to spend to much time on this part. There is more important code to work on.